src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PersonInfo } from '../PersonInfo/PersonInfo';
+import { getPeopleWithParents } from '../../helpers/helpers';
+
+export const PeopleTable = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  const loadPeople = async () => {
+    setIsLoading(true);
+    setIsError(false);
+
+    try {
+      const peopleFromServer = await getPeople();
+      const peopleWithParents = getPeopleWithParents(peopleFromServer);
+
+      setPeople(peopleWithParents);
+    } catch {
+      setIsError(true);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    loadPeople();
+  }, []);
+
+  const noPeopleOnServer = !people.length && !isLoading;
+
+  return (
+    <div className="block">
+      <div className="box table-container">
+        {isLoading && (
+          <Loader />
+        )}
+
+        {isError && (
+          <p data-cy="peopleLoadingError" className="has-text-danger">
+            Something went wrong
+          </p>
+        )}
+
+        {noPeopleOnServer && (
+          <p data-cy="noPeopleMessage">
+            There are no people on the server
+          </p>
+        )}
```

Comment:
Let's show only one error, it's obvious that when there will be an error - there will be no data from server (second message is about case when request was successful, but data wasn't received)
![Screenshot 2023-04-19 at 15 15 24](https://user-images.githubusercontent.com/84275496/233071904-516f235c-2103-4db6-86e6-8ef3b80698bd.png)